tests/functional/api/BackbeatServer.js

```diff
@@ -543,9 +556,9 @@ describe('Backbeat Server', () => {
 
                 assert(res.failures.description);
                 // Failures count = OPS_FAIL
-                assert.equal(res.failures.results.count, 150);
+                assert.equal(res.failures.results.count, 1);
```

Comment:
the value for OPS_FAIL and BYTES_FAIL have been changed (no longer matching the values set in `before`), perhaps comment needs updating?